### PR TITLE
Update owner files

### PR DESCRIPTION
**Pull request by skonto**
# Changes

-  Added the initial owners of this repo. The new approvers group was added in https://github.com/knative/community/pull/1562.
- Similar to https://github.com/knative-extensions/serving-progressive-rollout/pull/1 we need an update. 
- I will do a PR to knobots shortly to get the future updates.

/kind <kind>

Fixes #

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -5,6 +5,7 @@ approvers:
 - technical-oversight-committee
 - serving-wg-leads
 - serving-writers
+- autoscaler-keda-approvers
 
 reviewers:
 - serving-reviewers
```
Comment (skonto) [inline OWNERS:11]: I could replace this with `autoscaler-keda-approvers` to avoid noise but also it might be useful if others want to review stuff.

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

**Pull request closed without merging**